br/pkg/lightning/restore/check_template.go

```diff
@@ -82,7 +82,7 @@ func (c *SimpleTemplate) Collect(t CheckType, passed bool, msg string) {
 }
 
 func (c *SimpleTemplate) Success() bool {
-	return c.warnFailedCount+c.criticalFailedCount == 0
+	return c.criticalFailedCount == 0
```

Comment:
> // If the performance is not as expect or one of critical check not passed. it will stop import task.
> Collect(t CheckType, passed bool, msg string)

If `c.warnFailedCount > 0`,  the import task should also be failed. @3pointer  PTAL